src/pages/common/model_page.h

```diff
@@ -21,4 +21,10 @@ struct model_page {
     enum ModelType modeltype;
 };
 
+#if HAS_SCANNER
+#define PROTOCOL_SCANNERS 1
```

Comment:
PROTOCOL_SCANNERS always equals to HAS_SCANNER. why we need another constant?

Reply:
I planned on increasing PROTOCOL_SCANNERS to 2 when adding cc2500 scanner module.

Reply:
you're right, I removed it.